gateway/src/services/evm.nonce.ts

```diff
@@ -18,14 +17,6 @@ export class EVMNonceManager {
   // this should be private but then we cannot mock it
   public _provider: ethers.providers.Provider | null = null;
 
-  public static getInstance(): EVMNonceManager {
-    if (!EVMNonceManager._instance) {
-      EVMNonceManager._instance = new EVMNonceManager();
-    }
-
-    return EVMNonceManager._instance;
-  }
-
   // init can be called many times and generally should always be called
   // getInstance, but it only applies the values the first time it is called
   public async init(
```

Comment:
Since we're now giving different networks (e.g. Avalanche) their own `EVMNonceManager` - we should add the network name to the nonce manager init function.